build/emlib.rs

```diff
@@ -1,5 +1,5 @@
 #![deny(warnings)]
-#![feature(core, io, old_path)]
+#![feature(fs, io, old_path)]
```

Comment:
Remove fs-feature again. 
I'm also okay with temporarily allowing warnings in the build-script.